.drone.yml

```diff
@@ -3,31 +3,35 @@ kind: pipeline
 name: default
 
 services:
-  - name: docker
-    image: docker:19.03.0-rc3-dind
-    privileged: true
-    command:
-      - --dns=8.8.8.8
-      - --dns=8.8.4.4
-      - --mtu=1440
-    volumes:
-      - name: docker-socket
-        path: /var/run
+- name: docker
+  image: docker:19.03-dind
+  entrypoint:
+  - dockerd
+  command:
+  - --dns=8.8.8.8
+  - --dns=8.8.4.4
+  - --mtu=1440
+  - --log-level=error
+  privileged: true
+  volumes:
+  - name: dockersock
+    path: /var/run
 
 steps:
   - name: build-pull-request
     image: autonomy/build-container:latest
     pull: always
     commands:
+      - until docker ps; do sleep 1; done
       - docker buildx create --driver docker-container --platform linux/amd64 --name local --use
       - bldr pack --progress=plain -o autonomy -f pkg.yaml
+    volumes:
```

Comment:
Similar to all the comments above, this doesn't align with the other packages in this repo. You know me and consistency :)